modules/configure-proxy-on-an-existing-deployment.adoc

```diff
@@ -0,0 +1,37 @@
+// Module included in the following assemblies:
+//
+// * configuration/configure-proxy.adoc
+:_module-type: PROCEDURE
+[id="configure-proxy-on-an-existing-deployment_{context}"]
+= Configuring proxy on an existing deployment
+
+To configure the proxy in an existing deployment, you must export the `proxy-config` secret as a YAML file, update your proxy configuration in that file, and upload it as a secret.
+
+.Procedure
+
+. Save the existing secret as a YAML file:
++
+[source,terminal]
+----
+$ oc -n stackrox get secret proxy-config \
+  -o go-template='{{index .data "config.yaml" | \
+  base64decode}}{{"\n"}}' > tmp/proxy-config.yaml
+----
+. Edit the fields you want to modify in the YAML configuration file, as specified in the Configure proxy during installation section.
+//TODO Link to Configure proxy during installation section
+. After you save the changes, run the following command to replace the secret:
++
+[source,terminal]
+----
+$ oc -n stackrox create secret generic proxy-config \
+  --from-file=config.yaml=/tmp/proxy-config.yaml -o yaml --dry-run | \
+  oc label -f - --local -o yaml app.kubernetes.io/name=stackrox | \
+  oc apply -f -
+----
++
+[NOTE]
+====
+* You must wait for at least 1 minute, until {ocp} propagates your changes to Central and Scanner.
+* If you see any issues with outgoing connections after changing the proxy configuration, you must restart your Central and Scanner pods.
+====
+//TODO add link to restarting Central and Scanner
```

Comment:
You could always add this in an Additional resources section of the assembly for the time being, if you wanted to. Not necessary, though.

Reply:
Right now I want to keep it as it is.